scripts/setup_bls_build_flags.sh

```diff
@@ -35,10 +35,3 @@ case $OS in
       export DYLD_FALLBACK_LIBRARY_PATH=$LD_LIBRARY_PATH
       ;;
 esac
```

Comment:
why remove this?

Reply:
source scripts/setup_bls_build_flags.sh   will fail if not removing this

Reply:
I found this bug when I start a new node, because need to source bls library

Reply:
It can be fixed using "-v" I think.  Don't just remove it.